lib/LMSManagers/LMSEventManager.php

```diff
@@ -47,14 +47,17 @@ public function EventAdd($event)
             SYSLOG::RES_ADDRESS => empty($event['address_id']) ? null : $event['address_id'],
             SYSLOG::RES_NODE => $event['nodeid'],
             SYSLOG::RES_TICKET => empty($event['ticketid']) ? null : $event['ticketid'],
+            'netnodeid' => empty($event['netnodeid']) ? '' : $event['netnodeid'],
+            'netdevid' => empty($event['netdevid']) ? '' : $event['netdevid'],
         );
```

Comment:
Dlaczego nie jako "klucze"? Dalej nie patrzę :)

Reply:
i do tego pola typu ID jako `''`?

Reply:
To o takie klucze Ci chodzi (poprzednio myślałem że chodzi o dodatkowe INDEXy w DB w celu uzyskania wyższego performance przy wybieraniu)
https://github.com/chilek/lms/pull/2273/commits/b7b1b13c1d7289114ef70ac4c7d797f92dccf24f

Zrobione.